app/actions/page.ts

```diff
@@ -1,18 +1,19 @@
-import _ from 'lodash';
-import type { ThunkAction } from 'redux-thunk';
+import _, { find, map } from 'lodash';
```

Comment:
As we are started using not explicit imports (with `_`), we may try to migrate to the explicit ones 🎉 

Lodash methods used inside this file are:
- `each`
- `compact`
- `find`
- `map`

```suggestion
import { each, compact, find, map } from 'lodash';
```

WDYT? 🤔

Reply:
Done